lib/stream.js

```diff
@@ -42,6 +44,12 @@ const Stream = module.exports = require('internal/streams/legacy').Stream;
 Stream.isDisturbed = utils.isDisturbed;
 Stream.isErrored = utils.isErrored;
 Stream.Readable = require('internal/streams/readable');
+for (const key of ObjectKeys(operators)) {
+  const op = operators[key];
+  Stream.Readable.prototype[key] = function(...args) {
+    return Stream.Readable.from(ReflectApply(op, this, arguments));
```

Comment:
why `arguments` if we are using `args` before?

Reply:
Not sure... it was one of the suggestions.